deploy/modules/aws/tidb-operator/main.tf

```diff
@@ -1,6 +1,7 @@
 module "eks" {
   source = "terraform-aws-modules/eks/aws"
 
+  version            = "5.1.0"
```

Comment:
what is this version?

Reply:
This is the version for module terraform-aws-modules/eks/aws.
Each provider and the non-local module is recommended to set the version explicitly: 
https://www.terraform.io/docs/configuration/modules.html#module-versions
https://www.terraform.io/docs/configuration/providers.html#version-provider-versions